crates/vhost/src/backend.rs

```diff
@@ -65,6 +65,7 @@ impl VringConfigData {
 
 /// Memory region configuration data.
 #[derive(Default, Clone, Copy)]
+#[repr(packed)]
```

Comment:
The commit message doesn't match the code.
And please add a body to your commit message to explain why this change is needed.